src/containers/shared/components/Transaction/NFTokenCancelOffer/parser.ts

```diff
@@ -0,0 +1,21 @@
+import { NFTokenCancelOffer, NFTokenCancelOfferInstructions } from './types'
+import { TransactionParser } from '../types'
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+export const parser: TransactionParser<
+  NFTokenCancelOffer,
+  NFTokenCancelOfferInstructions
+> = (_, meta) => {
+  const cancelledOffers = meta.AffectedNodes.filter(
```

Comment:
Not for this PR but the case when the transaction fails cancelled offers wont be populated but the offer ids will have been supplied.

Reply:
There should be a test case for this scenario too